build.sbt

```diff
@@ -201,6 +201,7 @@ lazy val kernelApi = (project in file("kernel/kernel-api"))
     commonSettings,
     scalaStyleSettings,
     javaOnlyReleaseSettings,
+    Test / javaOptions ++= Seq("-ea"),
```

Comment:
enable asserts (i.e. in non-test code, like `LogReplay.java`) only while running tests.